### ISSUE TITLE
Plan/summary/diff count is inconsistent

### ISSUE THREAD

**Issue by d4nyll** [hashicorp/terraform-cdk#1547]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
- Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
- If you are interested in working on this issue or have submitted a pull request, please leave a comment

### cdktf & Language Versions

```
$ terraform --version
Terraform v1.1.4
on darwin_amd64

$ cdktf --version
0.9.0
```
Using TypeScript.

```
// package.json
  "dependencies": {
    "@cdktf/provider-aws": "^5.0.0",
    "cdktf": "^0.9.0",
    "constructs": "^10.0.45"
  },
```

### Actual Behavior

I changed the launch type of an AWS ECS Service. The Terraform plan shows `1 to add, 0 to change, 1 to destroy`

```
[2022-02-07T22:30:39.590] [DEBUG] default -   # aws_ecs_service.service must be replaced
-/+ resource "aws_ecs_service" "service" {
      - health_check_grace_period_seconds  = 0 -> null
      ~ iam_role                           = "aws-service-role" -> (known after apply)
      ~ id                                 = "arn:aws:ecs:us-east-1:<ID>:service/main/<NAME>" -> (known after apply)
      ~ launch_type                        = "EC2" -> "FARGATE" # forces replacement
        name                               = "<NAME>"
      + platform_version                   = (known after apply)
      - propagate_tags                     = "NONE" -> null
      - tags                               = {} -> null
      ~ tags_all                           = {} -> (known after apply)
        # (9 unchanged attributes hidden)

      - deployment_circuit_breaker {
          - enable   = false -> null
          - rollback = false -> null
        }

      - deployment_controller {
          - type = "ECS" -> null
        }

        # (2 unchanged blocks hidden)
    }

Plan: 1 to add, 0 to change, 1 to destroy.
```
But the CDKTF summary before I entered `yes` says `Diff: 0 to create, 0 to update, 1 to delete.`

After CDKTF is done, it says `Summary: 1 created, 0 updated, 0 destroyed.`

### Expected Behavior

Terraform plan output should be `Plan: 1 to add, 0 to change, 1 to destroy.`
CDKTF diff output should be `Diff: 1 to create, 0 to update, 1 to delete.`
CDKTF summary output should be `Summary: 1 created, 0 updated, 1 destroyed.`

### Steps to Reproduce

Create an ECS Service using CDKTF and change its launch type. I think any resources that requires a replacement (destroy + recreate) will surface the issue. I have seen the same behavior in replacing a task definition too.

**Comment by ansgarm** [comment]
Hi @d4nyll 👋
Thank you for reporting this issue!
This seems similar to #1269 and could be solved by #114.

**Comment by DanielMSchmidt** [comment]
We removed this view in favour of the terraform logs, so we can close this thanks to #1596
